src/routes/safe/components/Apps/index.jsx

```diff
@@ -113,10 +125,84 @@ function Apps({
 
   const iframeRef = useCallback((node) => {
     if (node !== null) {
-      setframeEl(node)
+      setIframeEl(node)
     }
   }, [])
 
+  const onSelectApp = (appId) => {
+    setAppIsLoading(true)
```

Comment:
When selecting the same app again this triggers and endless loading spinner (I think the error is not here, but that is were the loading spinner is set)

Reply:
fixed